src/services/doppler-api-client.test.ts

```diff
@@ -531,4 +533,99 @@ describe('HttpDopplerApiClient', () => {
       expect(result.success && result.value).not.toBe(null);
     });
   });
+
+  describe('getSubscriberFieldHistory', () => {
+    it('should get an error when the response is not valid', async () => {
+      // Arrange
+      const request = jest.fn(async () => {}); // Invalid response
+      const dopplerApiClient = createHttpDopplerApiClient({ request });
+
+      // Act
+      const result = await dopplerApiClient.getSubscriberFieldHistory({
+        subscriberEmail: 'a@a.com',
+        fieldName: 'MiPermiso',
+      });
+
+      // Assert
+      expect(request).toBeCalledTimes(1);
+      expect(result).not.toBe(undefined);
+      expect(result.success).toBe(false);
+    });
+
+    it('should call with the right parameters and parse successful response', async () => {
+      // Arrange
+      const subscriberEmail = 'a@a.com';
+      const fieldName = 'MiPermiso';
+      const fieldsResult = {
+        data: {
+          items: [
+            {
+              subscriberEmail: 'email@a.com',
```

Comment:
Here shouldn't be the `subscriberEmail`--> `a@a.com` instead of `email@a.com`?

Reply:
Yes, but it is the same, right?

I mean, what does the server return in `subscriberEmail` and `fieldName` is the server's responsibility. Right?

Reply:
Yes, but it's confusing to read, I'm questioning you because of this. Imagine ourselves in the future :trollface:

Reply:
Ok, thanks, changing...